Tweak gum mechanic

T-Nugget opened the pull request:
The jump counter now resets when player is on the floor.
The bubble now automatically pops if the player touches the floor.
Added the stalactite which pops the bubble.

TriOmega commented:
In future I would also include a change like adding CineMachine, but otherwise great work on this!

T-Nugget committed:
Tweak gum mechanic

The jump counter now resets when player is on the floor.
The bubble now automatically pops if the player touches the floor.
Added the stalactite which pops the bubble.
